src/risk_atlas_nexus/blocks/risk_mapping/risk_mapper.py

```diff
@@ -103,8 +108,56 @@ def generate(self, new_risks: List[Risk], existing_risks: List[Risk], inference_
                 mappings.append(mapping)
 
 
-        elif mapping_method == MappingMethod.RITS_INFERENCE:
-            pass
+        elif mapping_method == MappingMethod.INFERENCE:
+            # this query is just using name and description, not any other attributes 
+            usecases = [("ID: " + nr.id + ", Name: " + nr.name + ", Description: " + nr.description) for nr in new_risks]
+            
+            # using a prompt without the template examples, as they maybe do not exist for all taxonomies 
+            prompts = [
+                self.inference_engine.prepare_prompt(
+                    prompt_template=RISK_IDENTIFICATION_TEMPLATE,
```

Comment:
FYI: I have a robust way of getting a risk list through function calling and structure ouput. Already, implemented in Agentic AI governance. I will move the code to Risk Atlas sometime this or next week.